spec/spec_helper.rb

```diff
@@ -1,5 +1,14 @@
+require 'simplecov'
 require 'coveralls'
-Coveralls.wear_merged! 'rails'
+
+SimpleCov.formatters = [
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+
+SimpleCov.start 'rails' do
+  add_filter 'app/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.